src/cli.ts

```diff
@@ -1013,28 +1010,19 @@ function renew(network: CLINetworkAdapter, args: string[]) : Promise<string> {
         numOwnerUTXOs + numPaymentUTXOs - 1);
     });
 
-  const zonefileHashPromise = new Promise((resolve : any, reject : any) => {
-    if (!!zonefile) {
-      const sha256 = bitcoin.crypto.sha256(new Buffer(zonefile));
-      const h = (new RIPEMD160()).update(sha256).digest('hex');
-      resolve(h);
-    } else if (!!zonefileHash || blankZonefileHash) {
-      // already have the hash 
-      resolve(zonefileHash);
-    } else {
-      return getLastZonefileHash(network, name)
-        .then((zfh : string) => resolve(zfh))
-        .catch((e : Error) => reject(e));
-    }
-  })
-    .catch((e : Error) => {
-      console.error(e);
-    });
+  let zfh: string;
```

Comment:
@jcnelson The linter was not happy with this promise construction -- I refactored but not sure about if the error handling control flow is totally correct

Reply:
Will the call to `await getLastZonefileHash(network, name);` below cause this whole method to fail if it fails?  This is the intended behavior.

Reply:
Yes, sounds correct then